app/Views/components/layout.php

```diff
@@ -7,7 +7,7 @@
     <title>Más Noticias</title>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
```

Comment:
```suggestion
    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
```
Tienes una query string sin un valor y eso generá un fallo al pedir el recurso.